packages/babel-preset-es2015/src/index.js

```diff
@@ -73,3 +53,28 @@ function preset(context, opts) {
   };
 }
 
+
+/**
+ * This preset was originally an object, before function-based configurable presets were introduced.
+ * For backward-compatibility with anything that may have been loading this preset and expecting
+ * it to be a simple Babel config object, we maintain the old config here.
+ */
+const thePreset = preset({});
+
+export const plugins = thePreset.plugins;
+
+//For backward compatibility with babel-core < v6.13.x, we use the 'buildPreset' property
+//of the preset object for the preset creation function.
+export const buildPreset = preset;
+
+//I don't think it is necessary to make `buildPreset` non-enumerable as this version
+//of this preset is shipped only with newer versions of babel-core so will comment
+//out noting that all tests pass without it.
```

Comment:
Coincidentally I went to sleep last night thinking about this very thing. (Just woke and its 6am Satuday morning).  Will mull over coffee :-)